crates/loam-cli/src/commands/build/mod.rs

```diff
@@ -125,10 +131,12 @@ impl Cmd {
             ));
             cmd.arg("--crate-type=cdylib");
             cmd.arg("--target=wasm32-unknown-unknown");
-            if self.profile == "release" {
+            let profile = self.profile.clone().unwrap_or("release".to_string());
+            if profile == "release" {
                 cmd.arg("--release");
+            } else if profile == "debug" {
             } else {
-                cmd.arg(format!("--profile={}", self.profile));
+                cmd.arg(format!("--profile={profile}"));
             }
```

Comment:
if it is neither `release` nor `debug` then we ignore it?

Reply:
No we ignore if it is debug. Or said another way don't ignore it if isn't debug.